db/migrate/20211117195121_remove_destroyed_help_texts.rb

```diff
@@ -28,14 +28,14 @@
 
 class RemoveDestroyedHelpTexts < ActiveRecord::Migration[6.1]
   def up
-    custom_field_ids = CustomField
-      .pluck(:id)
-      .map { |id| "custom_field_#{id}" }
-
-    AttributeHelpText
-      .where("attribute_name LIKE 'custom_field_%'")
-      .where.not(attribute_name: custom_field_ids)
-      .destroy_all
+    execute <<~SQL
+      DELETE FROM attribute_help_texts
+      WHERE attribute_name LIKE 'custom_field_%'
+        AND attribute_name NOT IN (
+          SELECT CONCAT('custom_field_', id)
+          FROM custom_fields
+          )
+    SQL
```

Comment:
So it no longer calls any model and doesn't blow up because of the undefined enum column that Rails 7 complains about?

Reply:
Yes exactly that